plugins/services/src/js/containers/service-configuration/ServiceConfiguration.js

```diff
@@ -209,18 +206,15 @@ class ServiceConfiguration extends mixin(StoreMixin) {
     const { errors, service } = this.props;
     const { selectedVersionID } = this.state;
     const config = service.getVersions().get(selectedVersionID);
-    let content = null;
 
     if (config == null) {
-      content = <Loader />;
-    } else {
-      content = <ServiceConfigDisplay appConfig={config} errors={errors} />;
+      return <Loader />;
```

Comment:
Let's wrap the loader in a `<div className="container">` for consistency just like it was when it was in the render method.